doc/nrf/ug_nrf5340.rst

```diff
@@ -81,41 +81,51 @@ The OpenAMP library uses the IPM SHIM layer, which in turn uses the IPC driver i
 Available samples
 *****************
 
-nRF5340 samples consist of two separate images: one that runs on the network core and one that runs on the application core.
+nRF5340 samples usually consist of two separate images: one that runs on the network core and one that runs on the application core.
+
+Most samples that support nRF5340 are not dedicated exclusively to this device, but can also be built for other chips that have the radio peripheral (for example, the nRF52 Series).
+The following subsections give a general overview of what samples can run on nRF5340.
 
 Network samples
 ===============
 
```

Comment:
I changed this section to text instead of a list because it was pointing to one specific sample and a set of samples, but in a list, all items should be similar.
Like this, it is more consistent with the Application samples section as well.